src/components/filtermenu/filtermenu.js

```diff
@@ -35,15 +46,14 @@ define(['require', 'dom', 'focusManager', 'dialogHelper', 'loading', 'apphost',
         elem.querySelector('.filterOptions').innerHTML = html;
     }
 
-    function renderDynamicFilters(context, result, options) {
-        renderOptions(context, '.genreFilters', 'chkGenreFilter', result.Genres, function (i) {
+    renderDynamicFilters(context, result, options) {
+        this.renderOptions(context, '.genreFilters', 'chkGenreFilter', result.Genres, function (i) {
```

Comment:
If you leave the functions that wern't previously part of ` FilterMenu.prototype` you won't need to add `this.` when the function is called

Reply:
Got it. Just pushed those changes.

Reply:
Cool thanks